recipes/ceres-solver/all/conanfile.py

```diff
@@ -73,51 +66,88 @@ def configure(self):
+            self._cmake.definitions["CXX11_THREADS"] = self.options.use_CXX11_threads
+            self._cmake.definitions["CXX11"] = self.options.use_CXX11
+        self._cmake.definitions["SCHUR_SPECIALIZATIONS"] = self.options.use_schur_specializations
+        if self.settings.compiler == "Visual Studio":
+            self._cmake.definitions["MSVC_USE_STATIC_CRT"] = str(self.settings.compiler.runtime) == "MT" or str(self.settings.compiler.runtime) == "MTd"
+        self._cmake.configure()
+        return self._cmake
+
     def build(self):
-        #Make sure that cmake finds gflags is use_gflags=True
-        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                              "find_package(Gflags)",
-                              "find_package(Gflags REQUIRED)")
-        #On windows the library names can be gflags.dll or gflags_static.lib
-        tools.replace_in_file(os.path.join(self._source_subfolder, "cmake", "FindGflags.cmake"),
-                              "find_library(GFLAGS_LIBRARY NAMES gflags",
-                              "find_library(GFLAGS_LIBRARY NAMES gflags gflags_static")
-        for patch in self.conan_data["patches"][self.version]:
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
         cmake = self._configure_cmake()
         cmake.build()
 
     def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
         cmake = self._configure_cmake()
         cmake.install()
-        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "CMake"))
 
     def package_info(self):
+        libsuffix = ""
+        if self.settings.build_type == "Debug":
+            libsuffix = "-debug"
         self.cpp_info.names["cmake_find_package"] = "Ceres"
         self.cpp_info.names["cmake_find_package_multi"] = "Ceres"
-        self.cpp_info.components["ceres"].libs = tools.collect_libs(self)
+        self.cpp_info.components["ceres"].libs = ["ceres{}".format(libsuffix)]
         self.cpp_info.components["ceres"].includedirs = ["include", os.path.join("include","ceres")]
         if not self.options.use_glog:
             self.cpp_info.components["ceres"].includedirs.append(os.path.join("include","ceres", "internal", "miniglog"))
+        if self.settings.os == "Linux":
+            if self.options.get_safe("use_CXX11_threads", True):
+                self.cpp_info.components["ceres"].system_libs.append("pthread")
+        elif tools.is_apple_os(self.settings.os):
+            self.cpp_info.frameworks = ["Accelerate"]
```

Comment:
@prince-chrismc 
I guess so.
```suggestion
            if tools.Version(self.version) >= "2":
                self.cpp_info.frameworks = ["Accelerate"]
```